orttraining/orttraining/training_ops/cpu/aten_ops/aten_op_config.cc

```diff
@@ -14,21 +14,21 @@ namespace aten_ops {
 // We use regex to parse the strings, to make the parser simple, it requires some special formats
 // for these function strings, such as spaces in the strings.
 static const std::vector<std::pair<std::string, std::string>> ATEN_FUNCS = {
-    {"aten::embedding(Tensor<T> weight, Tensor<int64> indices, int padding_idx=-1, bool scale_grad_by_freq=False, bool "
-     "sparse=False) -> Tensor<T> result",
-     "aten::embedding_backward(Tensor<T> grad_result, Tensor<int64> indices, Tensor<T> weight, int padding_idx=-1, "
-     "bool scale_grad_by_freq=False, bool sparse=False) -> Tensor<T> grad_weight"},
-    {"aten::max_pool2d_with_indices(Tensor<T> self, int[] kernel_size, int[] stride, int[] padding=[0,0], int[] "
-     "dilation=[1,1], bool ceil_mode=False) -> (Tensor<T> output, Tensor<int64> indices)",
+    {"aten::embedding(Tensor<T> weight, Tensor<int64> indices, int padding_idx=-1, bool scale_grad_by_freq=False, "
+     "bool sparse=False) -> Tensor<T> result",
+     "aten::embedding_backward(Tensor<T> grad_result, Tensor<int64> indices, int<CPU> weight.size(0), "
```

Comment:
for the "int<CPU>" syntax, is this copied from pytorch, or something defined by ourselves?

Reply:
It's defined by me to distinguish the CPU input arguments and attribute arguments. As mentioned above, once we move all non-tensor arguments (attributes for now) to CPU input tensor inputs, we will not need this syntax.